### ISSUE TITLE
Rewards in a container

### ISSUE THREAD

**Issue by tass23** [Ravenwolfe/ServUO-XmlSpawner#2]
There is a bug that causes a client crash if you set the reward to visible (canseereward/true). If a player tries to open the bag that is visible in the questholder, crash. The work around has been to set canseereward/false. The bag is still visible in the questholder, but cannot be opened.

**Comment by Ravenwolfe** [comment]
You happen to have a crash log for that?

**Comment by tass23** [comment]
Nope, no crash log for the shard, it just crashes the client of the player that tries to open the reward bag. Arte was aware of it, but never got it patched.

**Comment by tass23** [comment]
I believe this is the issue with the crash right here, in QuestHolder.cs, part of the DisplayTo method:
`if (to.NetState != null && to.NetState.ContainerGridLines)
                to.Send(new ContainerContent6017(to, this));
            else
#endif
                to.Send(new ContainerContent(to, this));

            if (ObjectPropertyList.Enabled)
            {
                List<Item> items = this.Items;

                for (int i = 0; i < items.Count; ++i)
                    to.Send(((Item)items[i]).OPLPacket);
            }`
